apps/web/.env.example

```
@@ -54,6 +54,7 @@ B2C_CUSTOM_DOMAIN_PATH=
 # Template IDs are plain environment variables (not secrets)
 GOVUK_NOTIFY_TEMPLATE_ID_SUBSCRIPTION=template-uuid-here
 GOVUK_NOTIFY_TEMPLATE_ID_SUBSCRIPTION_PDF_ONLY=template-uuid-here
+GOVUK_NOTIFY_TEMPLATE_ID_SUBSCRIPTION_PDF_EXCEL=42f65ada-6de0-45da-822a-9632f6f682fd
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOVUK_NOTIFY_TEMPLATE_ID_SUBSCRIPTION_PDF_EXCEL key should go before the GOVUK_NOTIFY_TEMPLATE_ID_SUBSCRIPTION_PDF_ONLY key

(UnorderedKey)